spec/ruby/core/hash/shared/store.rb

```diff
@@ -22,15 +22,15 @@ def key.reverse() "bar" end
     k1.should_receive(:hash).and_return(0)
     k2.should_receive(:hash).and_return(0)
 
-    h[k1] = 1
-    h[k2] = 2
+    h.send @method, k1, 1
+    h.send @method, k2, 2
```

Comment:
Can you use parentheses here to be consistent with the other specs in this file?